pkg/kubectl/describe.go

```diff
@@ -1105,6 +1106,69 @@ func describeService(service *api.Service, endpoints *api.Endpoints, events *api
 	})
 }
 
+// EndpointsDescriber generates information about an Endpoint.
+type EndpointsDescriber struct {
+	client.Interface
+}
+
+func (d *EndpointsDescriber) Describe(namespace, name string) (string, error) {
+	c := d.Endpoints(namespace)
+
+	ep, err := c.Get(name)
+	if err != nil {
+		return "", err
+	}
+
+	events, _ := d.Events(namespace).Search(ep)
+
+	return describeEndpoint(ep, events)
+}
+
+func describeEndpoint(ep *api.Endpoints, events *api.EventList) (string, error) {
```

Comment:
This is minor, but can you change `describeEndpoint` to `describeEndpoints` since it's api.Endpoints not api.Endpoint?